phpdocker-io/readability-js-server
# Github actions improvements

**Changes:**

 * Upgrade all external steps to their latest versions
 * Ensure we docker build on all pull requests to master, but not push when not appropriate
 * Update step outputs from old to new style
